### ISSUE TITLE
Sync wrong when copy/pasting event in Outlook

### ISSUE THREAD

**Issue by Mi-Pf** [phw198/OutlookGoogleCalendarSync#2007]
**OGCS Version**: v2.11.0.0
**Edition**: Installed
**Sync Direction**: O<->G

**Bug Description**  
Nearly the same issue like #487.
When an already existing event is copied in Outlook and then pasted to another date, the original event will be deleted by OGCS.

**Steps to Reproduce the Issue**  
1. Create an event on any date and time.
2. Sync
3. Click on the event and press Ctrl+C.
4. Click on any other day in the calendar and press Ctrl+V. (change the time if needed, doesn't effect the outcome)
5. Sync
6. I will prompted by OGCS, if I wanted to delete the original event.

**Any other information**  
Windows 11 (64-bit) with Outlook 365 (Version 2410 | 64-bit)

**Comment by Mi-Pf** [comment]
Same behavior in **OGCS Version**: v2.11.1.0-alpha

**Comment by phw198** [comment]
[Documented here](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Limitations#outlook-appointment-copy--paste).

**Comment by Mi-Pf** [comment]
Thanks!

But OGCS is always running in the background. Is it enough when OGCS minimised in tray?

**Comment by MissMIsh** [comment]
Hi, I am experiencing a similar issue. whenever I Sync from Google to Outlook, it just automatically deletes the new diary entries i have - and there is no way to undo it. I cant figure out how to fix it. 

[outlook_appointments.csv](https://github.com/user-attachments/files/17758801/outlook_appointments.csv)
[outlook_appointments-prev.csv](https://github.com/user-attachments/files/17758804/outlook_appointments-prev.csv)
[outlook_create.csv](https://github.com/user-attachments/files/17758805/outlook_create.csv)

[OGcalsync.log](https://github.com/user-attachments/files/17758797/OGcalsync.log)
[OGcalsync_2024-11-14.log](https://github.com/user-attachments/files/17758799/OGcalsync_2024-11-14.log)

**Comment by phw198** [comment]
@Mi-Pf Yes, running in the background would be enough. The section in the wiki above in the link I previously posted also describes problems if a synced item is moved into or out of the synced date range - is that relevant to your issue?

@MissMIsh You mean manually created Outlook items are deleted by OGCS? That would be because "Merge items" is not turned on.
